Sources/UtiliKit/Container/Transitioning/ContainerTransitionContext.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  ContainerTransitionContext.swift
+//  UtiliKit-iOS
+//
+//  Created by Will McGinty on 1/22/19.
+//  Copyright © 2019 Bottle Rocket Studios. All rights reserved.
+//
+
+import UIKit
+
+/// This class is internal to the framework. It's concrete class type is never leaked and is instead exposed simply as an object conforming to `UIViewControllerContextTransitioning`. It's purpose is to function as a drop in replacement for the transition context objects vended by UIKit for presentations. Because the container utilizes it's own child-based transition environment, UIKit will not create a transition context for it's transitions. This context supports both interactive and animated transitions.
+class ContainerTransitionContext: NSObject {
+    
+    typealias Completion = (Bool) -> Void
+    
+    // MARK: Properties
+    var containerView: UIView
+    var percentComplete: CGFloat = 0
+    var presentationStyle: UIModalPresentationStyle
+    var transitionWasCancelled: Bool = false
+    var targetTransform: CGAffineTransform = CGAffineTransform.identity
```

Comment:
Additional `CGAffineTransform` not needed.
```swift
var targetTransform: CGAffineTransform = .identity
```

Reply:
Fixed